wlog/diaglog/diag1log/logger_default.go

```diff
@@ -28,17 +30,18 @@ func (l *defaultLogger) Diagnostic(diagnostic logging.Diagnostic, params ...Para
 }
 
 func toParams(diagnostic logging.Diagnostic, inParams []Param) []wlog.Param {
-	outParams := make([]wlog.Param, len(defaultTypeParam)+1+len(inParams))
-	copy(outParams, defaultTypeParam)
-	outParams[len(defaultTypeParam)] = wlog.NewParam(diagnosticParam(diagnostic).apply)
+	outParams := make([]wlog.Param, len(defaultTypeParams)+1+len(inParams))
+	copy(outParams, defaultTypeParams)
+	outParams[len(defaultTypeParams)] = wlog.NewParam(diagnosticParam(diagnostic).apply)
 	for idx := range inParams {
-		outParams[len(defaultTypeParam)+1+idx] = wlog.NewParam(inParams[idx].apply)
+		outParams[len(defaultTypeParams)+1+idx] = wlog.NewParam(inParams[idx].apply)
 	}
 	return outParams
 }
 
-var defaultTypeParam = []wlog.Param{
+var defaultTypeParams = []wlog.Param{
```

Comment:
Name should be kept as `defaultTypeParam` -- there's still only one param entry in the slice, and don't think that the change is significant enough to rename and introduce the new diff.